udev/sysconfig.issue-generator

```diff
@@ -1,9 +1,9 @@
 ## Path:        System/Login
 ## Description: /etc/issue generator
 ## Type:        string
-## Default:     "^[be]"
+## Default:     "^[beqichw]"
```

Comment:
I assume "q" is s390x?
And what are "i" and "h"?
"c" is the prefix for CNI (podman/kubernetes). We clearly don't want to whitelist that.
"w" is also used by kubernetes network...

Reply:
I only copied that from https://build.opensuse.org/package/view_file/Base:System/issue-generator/issue-generator.spec?expand=1 line 42 and added `w` at the end.

Reply:
Making that arch specific is wrong - is there not any way to distinguish "virtual" from "physical" interfaces?

Reply:
But interface names are arch specific. And "br" is also a "virtual" interface, not a "physical" one, but you still want that. But  no, with udev I'm not aware of a way to distinguish there. Another solution would be to not go via udev but write your own daemon, which can query the interface for all details.

Reply:
> But interface names are arch specific.

Technically interface names are arbitrary byte strings which don't have any meaning beyond what the user gives them, which doesn't make it simpler...

> But no, with udev I'm not aware of a way to distinguish there. Another solution would be to not go via udev but write your own daemon, which can query the interface for all details.

We could do that now already in place of the regex matching, though I don't have any idea what to check for directly.
An idea is to show everything which somehow in the tree (enslaved devices, vlan, etc.) ends up at a physical interface? Seems complex...

Reply:
> I will change the sysconfig file and spec file to add the "w".

+1 on that for now as that's the main issue currently